scripts/cangen/tests/test_traffic.py

```diff
@@ -0,0 +1,65 @@
+from cangen.traffic import calculate_bus_load, CANMessage
+import pytest
+
+
+def test_no_messages():
+    assert calculate_bus_load(
+        [
+            CANMessage(data_length=8, frequency=100),
+            CANMessage(data_length=5, frequency=50),
+        ],
+        500000,
+    ) == pytest.approx(3.60, rel=1e-2)
+
+    # zero frequency or negative frequency
+    assert calculate_bus_load(
```

Comment:
After adding the `raise ValueError` check mentioned earlier, you should verify that this function raises an error.

See <https://docs.pytest.org/en/7.1.x/how-to/assert.html#assertions-about-expected-exceptions>